src/docfx/restore/RestoreGitMap.cs

```diff
@@ -11,67 +11,77 @@ namespace Microsoft.Docs.Build
 {
     internal class RestoreGitMap : IDisposable
     {
+        private readonly string _docsetPath;
+        private readonly IReadOnlyDictionary<PackageUrl, DependencyGitLock> _dependencyGitLock;
         private readonly IReadOnlyDictionary<(PackageUrl packageUrl, string commit), (string path, DependencyGit git)> _acquiredGits;
 
-        public Dictionary<PackageUrl, DependencyGitLock> DependencyGitLock { get; private set; }
-
-        public RestoreGitMap(IReadOnlyDictionary<(PackageUrl packageUrl, string commit), (string path, DependencyGit git)> acquiredGits = null)
+        public RestoreGitMap(
```

Comment:
Make this constructor private and call `RestoreGitMap.Create` in `TocTest`?

Reply:
unfortunately `Create` does a lot extra logic like reading and serializing a file. That test badly needs improvements.

Reply:
I think we should handle `null` in `Create` instead